.changeset/silly-starfishes-share.md

```diff
@@ -0,0 +1,5 @@
+---
+"@yamada-ui/icon": patch
+---
+
+feat(icon): add `displayName` and `__ui` properties
```

Comment:
The correct term is `__ui__`, not `__ui`.

For unique terms, wrap the word with "`".
For example, "__ui__".